whois-commons/src/main/java/net/ripe/db/whois/common/etree/IntervalMap.java

```diff
@@ -161,4 +161,20 @@ public interface IntervalMap<K extends Interval<?>, V> {
      *         intervals.
      */
     List<V> findExactAndAllMoreSpecific(K key);
+
+    /**
+     * Finds all values associated with intervals that are contained within (more specific than)
+     * and whose prefix is higher, more specific
```

Comment:
Isn't all prefix in more specific will be higher than interval?  Should it be whose prefix is most higher ?

Reply:
changed